### DISCUSSION TITLE
【文档任务】【Obsidian 插件】介绍和活用

### DISCUSSION THREAD

**Discussion by juestchaos** [PKM-er/Pkmer-Docs#39] [Ideas]
## 下载量高的插件

- [ ] Annotator 的基本使用和活用范例
- [ ] Better Word Count  的基本使用和活用范例 @Quorafind
- [x] Dice Roller 的基本使用和活用范例  @Quorafind
- [x] Douban 的基本使用和活用范例
- [x] Hover Editor 的基本使用和活用范例 @juestchaos
- [x] obsidian minimal settings 的基本使用和活用范例  @juestchaos
- [ ] Note Refactor 的使用和活用范例
- [ ] Obsidian to Anki 的基本使用和活用范例
- [ ] Periodic Notes 的基本使用和活用范例
- [x] settings-search 的基本使用和活用范例 @juestchaos
- [x] Spaced Repetition 的基本使用和活用范例 @Quorafind 

### 2023-05-7—05-12
- [x] quick add @windily-cloud 
- [x] buttons @cumany 
- [ ] dataview @Quorafind 
- [x] Excalidraw @windily-cloud 
- [x] templater @cumany 
- [x] metaedit @cumany 
- [x] kanban @OS
- [x] task @cumany 
- [x] metadata menu  @宏沉
- [ ] quickswitcher@windily-cloud 

### AI 相关
- [x] Text Genererator  的基本使用和活用范例 @windily-cloud 

### 编辑类
- [x] pangu 的基本使用和活用范例 @juestchaos 
- [x] Easy Tpying 的基本使用和活用范例 @cumany 
- [x] Editor Syntax Highlight 的基本使用和活用范例 @juestchaos
- [x] comments 的基本使用和活用范例 @juestchaos
- [ ] Checklist 的基本使用和活用范例
- [x] Copy button for code blocks 的基本使用和活用范例 @juestchaos
- [x] cMenu Plugin 的基本使用和活用范例 @cumany 建议文末或者文前强烈引导给 editing toolbar
- [x] Creases 的基本使用和活用范例 @juestchaos
- [x] Admonition 的使用和活用范例 @cumany 【感觉可以扩展为会员文档】
- [x] Highlightr 的基本使用和活用范例 @juestchaos
- [x] latex-suite @windily-cloud 
- [x] Dynamic Highlights 的基本使用和活用范例 @Quorafind 

### 图片和图像
- [ ] Ozan's Image in Editor Plugin 的基本使用和活用范例

### 图标类
- [x] obsidian-icons-plugin 的基本使用和活用范例 @juestchaos 
- [x] obsidian-icon-shortcodes 的基本使用和活用范例 @juestchaos 
- [x] obsidian-emoji-toolbar 的基本使用和活用范例 @juestchaos
- [x] obsidian-icon-swapper 的基本使用和活用范例 @juestchaos
- [x] emoji-magic 的基本使用和活用范例 @juestchaos

### 思维导图类
- [x] obsidian-mindmap-nextgen 的基本使用和活用范例 @cumany 
- [x] obsidian-enhancing-mindmap 的基本使用和活用范例 @cumany 

### 日期类
- [x] Reminder 的基本使用和活用范例 @juestchaos
- [x] Natural Language Dates 的基本使用和活用范例  @juestchaos 

### 目录类
- [x] Obsidian Dynamic ToC 的基本使用和活用范例 @juestchaos 
- [x] Table of contents  的基本使用和活用范例 obsidian-plugin-toc @juestchaos 
- [x]  Obsidain 插件 Floating TOC 的基本使用和活用范例 obsidian-plugin-toc  @juestchaos @cumany (感觉 Cuman 会写的比我好很多)

### 任务类
- [ ] Task Progress Bar 的基本使用和活用范例 @Quorafind 
- [x] Kanban 的基本使用和活用范例 @OS
- [x] Day Planner 的基本使用和活用范例 @OS
- [ ] Obsidian Dynamic Timetable 的基本使用和活用范例

### 链接类
- [x] Supercharged Links 的基本使用和活用范例 @cumany 
- [x] Auto Link Title 的基本使用和活用范例 @juestchaos
- [x] url-into-selection 的基本使用和活用范例 @juestchaos 
- [x] external-favicon 的基本使用和活用范例 @juestchaos
- [x] link-favicon 的基本使用和活用范例 @juestchaos

### 标签类
- [x] tag wargle 的基本使用和活用范例 @juestchaos 

### 插入预览 和 第三方集成
- [x] Custom Frames 的基本使用和活用范例 @juestchaos 
- [x] surfing 的基本使用和活用范例 @Quorafind 
- [x] convert-url-to-iframe 的基本使用和活用范例 @juestchaos 
- [x] Simple Embeds 的基本使用和活用范例 @juestchaos 
- [ ] mx-bili-plugin 的基本使用和活用范例  @juestchaos 
- [ ] media-extended 的基本使用和活用范例  @juestchaos 
- [x] Rich Links 的基本使用和活用范例 @juestchaos
- [x] link-embed 的基本使用和活用范例
- [x] auto-card-link 的基本使用和活用范例
- [ ] Obsidian-links 的基本使用和活用范例

### 表格类
- [x] Sortable 的基本使用和活用范例 @juestchaos 
- [x] Advanced Table 添了公式用法 @windily-cloud 
- [x] Excel to Markdown Table 的基本使用和活用范例 @cumany

### 文件管理类
- [x]  file-tree-alternative 的基本使用和活用范例 @juestchaos 
- [x] Quick Explorer 的基本使用和活用范例 @juestchaos
- [x] novel-word-count 的使用和活用范例 @juestchaos
- [x] Collapse All 的基本使用和活用范例 @juestchaos
- [x] tagfolder 的基本使用和活用范例 @juestchaos
- [x] cycle-through-panes 的基本使用和活用范例 @cumany
- [x] obsidian-tabs的基本使用和活用范例 @juestchaos

### 天气
- [x] weather-fetcher 的基本使用和活用范例 @juestchaos

### markdown to ppt
- [ ]  obsidian-marp-slides 的使用和活用范例 【感觉可以扩展为会员文档】
- [ ]  advanced slide 的使用和活用范例 【感觉可以扩展为会员文档】

### 美化
- [x] obsidian-day-and-night 的基本使用和活用范例 @juestchaos
- [x] obsidian-list-callouts 的基本使用和活用范例 @juestchaos

## 新发布，值得关注的插件
- [x] Heading Level Indent 的基本使用和活用范例 @juestchaos
- [x] ProgressBar 的基本使用和活用范例 @juestchaos

**Comment by jerrywcy** [comment]
补充：
- [ ] 【Obsidain 插件】[Path Finder](https://github.com/jerrywcy/obsidian-path-finder) 的基本使用和活用范例

**Comment by aiyou9** [comment]
# 任务类

- [x] 【Obsidain 插件】[Tasks Calendar Wrapper](https://github.com/Leonezz/obsidian-tasks-calendar-wrapper) 任务的时间线和日历视图的基本使用和活用范例 @aiyou9

**Comment by aiyou9** [comment]
-  [x] 【Obsidain 插件】[Obsidian Quiet Outline](https://github.com/guopenghui/obsidian-quiet-outline) obsidain 大纲插件的基本使用和活用范例 @aiyou9 
juestchaos 已经写了，感谢os大佬

**Reply by windily-cloud** [reply]
666啊，大佬

**Reply by aiyou9** [reply]
> 666啊，大佬

🤝我这只是搞搞自己感兴趣插件的教程，
真正的大佬，是写上面列表中插件教程的大佬👍

**Reply by juestchaos** [reply]
已经合并，感谢社区做贡献

**Comment by juestchaos** [comment]
- [x] pandoc  @二宝

**Comment by juestchaos** [comment]
- [x] bookmark @ Huajin

**Comment by juestchaos** [comment]
- [ ] OZ Calendar Plugin 的基本使用和活用范例
- [ ] colorful tag 的基本使用和活用范例
- [ ] DB Folder 的基本使用和活用范例
- [ ] core search assistant 的基本使用和活用范例

**Comment by juestchaos** [comment]
- [x] canvas presentation @PandaNocturne #271 
- [x] Excalidraw插件slideshow  @PandaNocturne #271

**Comment by juestchaos** [comment]
- [ ] Awesome Image 的基本使用和活用范例 @tsaoshuang 
- [ ] Copy lmage and URL context image @calmwaves111
